### PR TITLE
Adds two charts with hard-coded data

### PR DESCRIPTION
**Pull request by nitinramadoss**
- Adds CSS style rules for box shadows
- Adds pie chart with hard-coded data
- Adds bar chart with hard-coded data

### CODE REVIEW

File: capstone/src/main/webapp/index.html [hunk 1]
```diff
@@ -31,13 +31,18 @@ <h2>Top Negative Associations:</h2>
             <li>Thing 3</li>
           </ul>
         </div>
-        <div class="stats-divider"></div>
         <div id="second-stats" class="container second-stats">
-          <p>Something here about stats</p>
+          <h2>Sentiment Percentages</h2>
+          <script src="https://www.gstatic.com/charts/loader.js"></script>
+          <script>
```
Comment (prashantiA) [inline capstone/src/main/webapp/index.html:37]: Can this script be combined into a single JS function that is called? Nitpick but would be cleaner than having a two line script in the HTML IMO

Comment (prashantiA) [reply capstone/src/main/webapp/index.html:37]: Just this one minor nitpick, other than that LGTM!

Comment (nitinramadoss) [reply capstone/src/main/webapp/index.html:37]: I am not sure if that's possible, the charts API docs recommend placing the script tags in the html file

Comment (prashantiA) [reply capstone/src/main/webapp/index.html:37]: Yeah its something worth looking into, personally I find it very messy to have js code in a script tag like that

**Comment by prashantiA** [comment]
LGTM!

File: capstone/src/main/webapp/index.html [hunk 2]
```diff
@@ -31,13 +36,13 @@ <h2>Top Negative Associations:</h2>
             <li>Thing 3</li>
           </ul>
         </div>
-        <div class="stats-divider"></div>
         <div id="second-stats" class="container second-stats">
```
Comment (lee12jin) [inline capstone/src/main/webapp/index.html:39]: For these id tags, you might want to make it more descriptive now that you're replacing the placeholders.

`<div id="sentiment-pie-stats"...`

File: capstone/src/main/webapp/index.html [hunk 3]
```diff
@@ -31,13 +36,13 @@ <h2>Top Negative Associations:</h2>
...
-          <p>Something here about stats</p>
+          <h2>Sentiment Percentages</h2>
+          <div id="sentiment-pie-chart"></div>
         </div>
-        <div class="stats-divider"></div>
         <div id="third-stats" class="container third-stats">
```
Comment (lee12jin) [inline capstone/src/main/webapp/index.html:43]: Same comment here.

File: capstone/src/main/webapp/script.js [hunk 4]
```diff
@@ -18,3 +18,37 @@ function addListElement(list, contents) {
...
+    stats.addColumn('string', 'Period');
+    stats.addColumn('number', 'Number of Responses');
+
+    stats.addRows([
+        ['Daily', 4],
+        ['Weekly', 15]
+    ]);
+
+    // Instantiate and draw the chart.
+    let chart = new google.visualization.BarChart(document.getElementById('response-bar-chart'));
```
Comment (jasperlee-google) [inline capstone/src/main/webapp/script.js:52]: Should thse `let` be `const` given the values aren't being re-assigned?

Comment (nitinramadoss) [reply capstone/src/main/webapp/script.js:52]: I'll make the change

**Pull request merged**